### DISCUSSION TITLE
Input text with audio

### DISCUSSION THREAD

**Discussion by DeafNet** [SubtitleEdit/subtitleedit#7279] [Q&A]
I was wondering if they have input text and audio by using whisper ai and getting srt ?

**Comment by niksedk** [comment]
SE can get an srt file from audio (or video with audio)... either via Whisper or via Vosk.

Try the latest beta version here: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip
(The beta version is preferred atm, at some model links have changed - use the Purfview Faster-Whisper)

It's in the video menu:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/523d48f4-fb23-455c-9932-6d3ca1ccaa4e)

Read more here: https://www.nikse.dk/subtitleedit/help#audio_to_text_whisper

**Reply by DeafNet** [reply]
I mean I add the breakdown text to the system and add audio so whisper can syncs into timestamp ?

**Reply by niksedk** [reply]
Whisper generates text and timestamps. SE/Whisper cannot sync an existing text.

**Reply by DeafNet** [reply]
> timestamps

I mean to said exit text and audio to create from whisper in timestamp. Is this possible ?
